docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -368,13 +356,13 @@ T1
 Source blocks:
 T1: B1, B2, B3
 
-Total indices size of all source blocks: 
+Total indices size of all source blocks:
 100G
 ```
 
-Partition Number = (100G / 64G) => round up to next 2^x = 2
+Partition Number = (100G / 64G = 1.5625) => round up to next 2^x = 2
 
-Partitioning: There is only one time interval from all source blocks which means it is compacting level 1 blocks. Partitioning needs to include all source blocks in each partitions.
+Partitioning: There is only one time interval from all source blocks which means it is compacting level 1 blocks. Partitioning needs to include all source blocks in each partition.
```

Comment:
Not sure if I get `time interval` correctly. You mean the block duration or time range of the block? Like 2h, 6h, etc

Reply:
That's right. I can update it to time range to be consistent.